language/tools/move-cli/tests/sandbox_tests/generate_struct_layout/args.txt

```diff
@@ -4,6 +4,11 @@ sandbox publish
 sandbox generate struct-layouts --module storage/0x00000000000000000000000000000001/modules/M2.mv --struct C --type-args u64
 sandbox generate struct-layouts --module storage/0x00000000000000000000000000000001/modules/M2.mv --struct C --type-args address
 sandbox generate struct-layouts --module storage/0x00000000000000000000000000000001/modules/M2.mv --struct C --type-args vector<u8>
+sandbox generate struct-layouts --module storage/0x00000000000000000000000000000001/modules/M2.mv --struct C --type-args u8
```

Comment:
This is a somewhat indirect way to test the type tag parser... I would suggest adding some new unit tests to the bottom of `parser.rs` instead

Reply:
Very true, thank you. Fixed now.
But something weird/unrelated is causing CI failures at build setup stage.

Reply:
Thanks for adding the unit tests! Would you mind deleting these other tests?

Reply:
Done.

Reply:
Done. Thanks for feedback. Still learning my way around the repo.